docs/source/operations/baf_verify.md

```diff
@@ -238,6 +238,41 @@ For final checking of the validity of the indy network.
 
 Upon successful connection, should display a `Pool Connected Successfully` msg.
 
+
+---  
+
+
+## R3 Corda Checks
+
+The flow chart shows the R3 Corda process. To verify the steps of deployment, follow the Verification Table 'R', to troubleshoot the general errors.
+
+![](./../_static/corda_flowchart.png)

```

Comment:
Flowchart should contain registration check(because without that the node will not progress).

Reply:
Added registration check to the flowchart